kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/JobServiceIntegrationTest.java

```diff
@@ -411,4 +366,30 @@ public void testScheduleSearchByCommandCancelJob() throws Exception {
         jobServicesClient.cancelRequest(jobId);
     }
 
+    private void assertRequestInfoInstance(RequestInfoInstance expected, RequestInfoInstance actual) {
+        assertNotNull(actual);
+        assertEquals(expected.getId(), actual.getId());
+        assertEquals(expected.getBusinessKey(), actual.getBusinessKey());
+        assertEquals(expected.getStatus(), actual.getStatus());
+        assertEquals(expected.getCommandName(), actual.getCommandName());
+    }
+
+    private RequestInfoInstance createExpectedRequestInfoInstance(Long jobId) {
```

Comment:
You could add a parameter here providing expected status value.
That way you would be able to reuse this for https://github.com/droolsjbpm/droolsjbpm-integration/pull/504/files#diff-95c03f7ac62da390d73cc97d48de876fR271